openstack/sharedfilesystems/v2/shares/results.go

```diff
@@ -144,3 +144,35 @@ func (r GetExportLocationsResult) Extract() ([]ExportLocation, error) {
 	err := r.ExtractInto(&s)
 	return s.ExportLocations, err
 }
+
+// GrantAccess contains all information associated with an OpenStack share Grant Access
+type GrantAccess struct {
```

Comment:
Following on from my comment above on `GrantAccessRequest`, response objects should be nouns. Here, something as simple as `Access` or `AccessRight` would work fine (again, admittedly confusing because "access" can be both a noun and a verb).

Perhaps @jtopjian has a better suggestion for the name